parser.y

```diff
@@ -112,7 +112,7 @@ import (
 	distinct		"DISTINCT"
 	distinctRow		"DISTINCTROW"
 	div 			"DIV"
-	doubleType		"DOUBLE"
+	doubleType		"DOUBLE / FLOAT8"
```

Comment:
> According to [documentation](https://www.gnu.org/software/bison/manual/html_node/Token-Decl.html#Token-Decl) in terms of `%token`,
> 
> > Once you equate the literal string and the token name, you can use them interchangeably in further declarations or the grammar rules. The yylex function can use the token name or the literal string to obtain the token type code number (see Calling Convention). Syntax error messages passed to yyerror from the parser will reference the literal string instead of the token name.
> 
> it's better not to make this change.

MySQL alias `FLOAT4` to `FLOAT`, `FLOAT8` to `DOUBLE` in its symbol map. See: [mysql-server/sql/lex.h](https://github.com/mysql/mysql-server/blob/817fcb883a8457fcf507156bb1a2052e887acc4c/sql/lex.h#L243).
So, there's no independent symbol for `FLOAT4` and `FLOAT8`, they are done in the lexer. I think it's better to follow `MySQL`'s behavior here. What do you think? @tangenta